carpool/templates/add.html

```diff
@@ -89,6 +96,22 @@ <h2>Add new ride</h2>
 
                     </div><!-- end .col-lg-12 -->
 
+                    <div class="messages">
+			              <font align = "center">
+                          {% if messages %}
+                           <img src = "{% static "images/success.gif" %}">                   
```

Comment:
The image is quite large and doesn't disappear automatically. Can you hide it after some time using jquery or just js?

Reply:
You can also use a smaller image.